tests/scripts/all.sh

```diff
@@ -4994,11 +4994,15 @@ config_block_cipher_no_decrypt () {
 }
 
 component_test_block_cipher_no_decrypt_aesni () {
+    # This consistently causes an llvm crash on clang 3.8, so use gcc
+    export CC=gcc
     config_block_cipher_no_decrypt 0
     common_block_cipher_no_decrypt
 }
 
 component_test_block_cipher_no_decrypt_aesni_use_psa () {
+    # This consistently causes an llvm crash on clang 3.8, so use gcc
+    export CC=gcc
```

Comment:
Exports aren't "local" to shell functions, right? So this means that any functions called after this one will have `CC` exported and set to `gcc`, until some other code changes it.

That means, depending on which functions are executed in which order, that different invocations of `all.sh` might use different compilers for different components...

Reply:
IIUC they are called via a sub process for this reason.

Reply:
Ah okay, sorry, of course